hedera-node/configuration/testnet/settings.txt

```diff
@@ -26,7 +26,7 @@ virtualMap.preferredFlushQueueSize,            10000
                                           #          Events           #
                                           #############################
 
-event.creation.maxCreationRate,                30
+event.creation.maxCreationRate,                20
```

Comment:
I think you missed the preprod settings.txt

Reply:
fixed